aya/src/bpf.rs

```diff
@@ -166,6 +166,10 @@ impl Features {
     }
 }
 
+/// It is unified to eBPF because there is a loss of whether it is BPF or eBPF because it is confused.
+#[deprecated]
```

Comment:
```suggestion
/// Deprecated: `BpfLoader` was renamed to `EbpfLoader` to avoid confusion between eBPF and classic BPF.
#[deprecated(note = "Use `EbpfLoader` instead")]
```

The